doc/neps/nep-0038-SIMD-optimizations.rst

```diff
@@ -8,7 +8,7 @@ NEP 38 — Using SIMD optimization instructions for performance
 :Status: Final
 :Type: Standards
 :Created: 2019-11-25
-:Resolution: https://mail.python.org/archives/list/numpy-discussion@python.org/thread/PVWJ74UVBRZ5ZWF6MDU7EUSJXVNILAQB/#PVWJ74UVBRZ5ZWF6MDU7EUSJXVNILAQB
+:Resolution: `NumPy Discussion <https://mail.python.org/archives/list/numpy-discussion@python.org/thread/FVKH4IWXYVZSTZEYKZJ7XWHU4EXX3YLS/>`_
```

Comment:
The link target here is not correct, it should be the same as in the removed line.